src/utils.rs

```diff
@@ -107,4 +137,17 @@ mod test {
             );
         }
     }
+
+    #[test]
+    fn test_normalize_unicode_chars() {
+        const TEST_MAP: [(&str, &str); 3] = [
+            (r#"“Hello, World”"#, r#""Hello, World""#),
```

Comment:
This test case doesn't feel very useful. Isn't everything it's checking already covered by the next one?

It might be good, though, to have a test case checking string without any normalization needed, or alternatively, check that the function is idempotent, i.e. applying it to the result yields the same result.